code/modules/paperwork/helpers.dm

```diff
@@ -0,0 +1,73 @@
+/**Tries to find a readily accessible pen in the user's held items, and in some of its inventory. */
+/proc/get_accessible_pen(var/mob/user)
```

Comment:
Why is this not a mob proc?

Reply:
Because why?
That's an awfully niche proc to have on the base mob?

Reply:
I mean, I don't really care that much where it is. It just sounded too niche of a functionality to put on the base mob.

Reply:
It's generally strictly preferable to write instance procs to global procs, as they can be overridden. For this proc, it seems logical to have it be on the instance, as different mob types can then cleanly override with pen-finding behavior specific to them (borg pen module, custom slots, pen organs, whatever).

Reply:
@afterthought2 
That's true. 
I'll figure out something.